episodes/03-create.md

```diff
@@ -17,12 +17,40 @@ exercises: 0
 
 ::::::::::::::::::::::::::::::::::::::::::::::::::
 
+## The `Git` jargon
+
+`Git` is a topic that contains a lot of words to do version control.
+
+![Word cloud for Git from <https://thoughtbot.com/blog/recommending-blog-posts>](fig/topic-git.png)
+
+We will locate them using this workflow bellow as template. We will relate Version control _actions_ that we can perform with specific _`git verb` commands_. These verbs will record your changes between `Git` _spaces_ associated to your folder.
+
+![Workflow will show actions, `git verb` commands, and spaces.](fig/cut-git-verb_map-00.png)
+
+In this episode, we are going to learn how to initialize Git to create a Local Repository in our folder, also known as Working directory or Workspace.
+
+![Initialize a Local Repository in your Workspace with the `git init` command verb](fig/cut-git-verb_map-01.png)
```

Comment:
This figure may be confusing as it could be misinterpreted as `git init` moving the "local repository" to a different place/space. I think it is fine to leave the figure in, but best to be aware that learners may assume something different.

Reply:
I agree to make this more explicit after showing the `.git` hidden folder https://github.com/epiverse-trace/git-rstudio-basics/pull/13/commits/b3443ddab7085a2602154f789b579e8eb2df7304